channels/channel_map.py

```diff
@@ -537,12 +540,19 @@ def buildHodoPosChannels(run=316):
     return hodoscope_channels
 
 
-def findTimeReferenceDelay(channel, run=1040):
-    if str(run) not in triggerdelay.keys:
+def findFanoutTimeReferenceDelay(channel, run=1040):
+    if str(run) not in triggerdelay.keys():
         return triggerdelay["default"][channel]
     else:
         return triggerdelay[str(run)][channel]
 
+def findDRSTriggerMap(channel, run=1040):
+    result = "_".join(channel.split("_")[:3])
+    print(triggermap.keys())
+    if str(run) not in triggermap.keys():
+        return triggermap["default"][result]
+    else:
+        return triggermap[str(run)][result]
```

Comment:
trigger is not always channel8? it depends on the runs?

Reply:
Trigger is always channel8 but maybe we want to use the MCP channel as the "trigger" sometimes. I just put the functionality in to do things run by run, just in case.